### PR TITLE
Adding fixed errors from Stylelint to support the CSS file

### PR DESCRIPTION
**Pull request by ReyAntonio12**
By mistake, I accepted all pull requests before.

Please look at this pull request instead.

Thank you.

### CODE REVIEW

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @ReyAntonio12 👍, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close

## Highlights
Setup linters correctly ✔️
Two files committed: .html and .css file ✔️
node_modules and test.md ignored by GitHub ✔️

## Required Changes ♻️

- [ ]  Kindly edit your PR title and description to be more detailed, it should be a summary of all you did in the PR; this will give the code reviewer(s) a glimpse of what your PR is all about. 👍

- [ ] Kindly ensure that your branch name is descriptive enough, this should give an overvew of all that you implemented in the branch. Kindly checkout to new branch and use the correct name and mention the old PR in the new PR.

- [ ] Kindly implement the following in your README file:

1. Remove or customize the microverse logo with yours if you have one. 👍 
2. Kindly ensure that all the mandatory headers are added (`Show your support` and `Acknowlegement`).
3. Ensure that all the sections in the Getting started header are added as well. ℹ️ 
4. Remove the placeholders from live demo and license sections. 👍 

Implementing all these will make your repo looks professional and more informative. 🙏 

## Optional suggestions
- [optional] I observed that the you've done almost all the project requirement in the main branch, this should not be for subsequent PR. Kindly take note that all project requirement for a milestone supposed to be implemented in the feature branch. This will allow the code reviewers to have full interaction with your PR. ℹ️  

- [optional] Kindly ensure that your commit messages are descriptive enough for subsequent PRs, this helps other developers understand the purpose of the commit. 👍 

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ReyAntonio12** [comment]
Hello, can you tell me what PR means? I would like to get to know more about that acronym and don't get confused anymore.

Also, as I leave in my first comment. I did by mistake a few pull requests and merged the pull request with the main branch when it was ready. I was working with my other branches.

**Comment by ReyAntonio12** [comment]
Please check the changes done based on your reviews. 

Thank you.

File: README.md [hunk 1]
```diff
@@ -68,17 +67,48 @@ div align="center">
   </ul>
 </details>
 
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```
Comment (Eternalgratis) [inline README.md:73]: - Well done for attempting the Microverse README template. However, there are still some broken links in your README. E.g The "back to top" link as highlighted isn't working. Kindly consider fixing it so you can have a professional Readme

**Review by Eternalgratis — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️
## Hi @ReyAntonio12 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but some are still not addressed yet.

almost-there

 ## Highlights
✅ You used the correct GitHub flow
✅ Descriptive pull request message
✅ linter error checks validated

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (ReyAntonio12) [reply README.md:73]: Fixed this issue, it should be working now.

**Review by shahadat3669 — APPROVED** [review]
Hi @ReyAntonio12 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Setup linters correctly ✔️
Two files committed: .html and .css file ✔️
Implemented changes as per Previous Reviewer. ✔️


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**